src/Polytopes/LinearProgram.jl

```diff
@@ -101,14 +169,65 @@ function maximal_vertex(lp::LinearProgram)
    end
 end
 
+
+"""
+    minimal_value(LP::LinearProgram)
+
+Return, if it exists, the minimal value of the objective function of LP over the feasible region
+   of LP. Otherwise, return `-inf`.
```

Comment:
```suggestion
Return, if it exists, the minimal value of the objective function of `LP` over the feasible region
of `LP`. Otherwise, return `-inf`.
```